ncm-metaconfig/src/main/metaconfig/ssh/pan/server_config.pan

```diff
@@ -4,11 +4,12 @@ include 'metaconfig/ssh/schema';
 
 bind "/software/components/metaconfig/services/{/etc/ssh/sshd_config}/contents" = sshd_config_file;
 
-prefix "/software/components/metaconfig";
+# since final locks the whole path, bind it to a fix value and set it as default too
+#    TODO: support in compiler
+bind "/software/components/metaconfig/commands/sshd_test_stdin" =
```

Comment:
So if a site wants to change this they need to patch the file rather than using e.g. site-config.pan?

Reply:
yes. i sort of see your point, but on the other hand also not. 

this is the `ncm-ssh` test code. you want to test the config in another way or something else enitrely, but if you know a better way to do so i'd like to hear about it.
but i guess this does not go well with eg OS changes.

i need to think about it. i would like that at least by default the templates provide something and that it is immutable. this is the closest i can think of right now.

if you have no real case, i'd like to keep it right now. PRs in future etc etc

Reply:
We currently run sshd from AFS not /usr/ so one of our major motivations to allow arbitrary commands is to allow us to customise this. So making it necessary to fork it to modify this seems a little odd. The default makes perfect sense, but activating this module is optional, same as using ncm-ssh is optional today.

At some point I envisage us moving to /usr/bin/sshd, at which point happy to see this enabled.

Reply:
@ned21 ok. i'll split the templates so it's easier to customise this bit.

Reply:
@ned21 giving it some more thought, i'm not going to change it.
easiest way to customize it actually is to register a new command, and overwrite the `actions/test` below.

if you customise things before this is included, i can make it a default assignment below.

Reply:
Good point - changing the config instead of changing the command makes a lot more sense.